fmriprep/workflows/util.py

```diff
@@ -174,7 +194,7 @@ def init_skullstrip_bold_wf(name='skullstrip_bold_wf'):
     return workflow
 
 
-def init_bbreg_wf(bold2t1w_dof, name='bbreg_wf'):
+def init_bbreg_wf(use_bbr, bold2t1w_dof, omp_nthreads, name='bbreg_wf'):
```

Comment:
An additional paragraph describing the feature being added would be very valuable. Just a couple of sentences to describe what happens if `use_bbr=None`.